src/apispec/core.py

```diff
@@ -47,6 +49,22 @@ def to_dict(self):
             if v != {}
         }
 
+    def example(self, name, component=None, **kwargs):
+        """Add a new example to the spec
+
+        :param str name: identifier by which example may be referenced.
+        :param dict component: schema definition.
+
+        https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#exampleObject
+        """
+        if name in self._examples:
+            raise DuplicateComponentNameError(
+                'Another example with name "{}" is already registered.'.format(name)
+            )
+        # Skipping plugin helper
```

Comment:
I'd remove this comment.

Reply:
Yes, I had this added so I can add the `example_helper` support in this same PR, however, this may extend to several discussions, hence it can just be added who are using this package without plugin support.

Since this PR makes much more sense to document *response schemas* (normally you may only want to add example per schema field, which is good for plugin like marshmallow), marshmallow fields may be overkill features. `FlaskPlugin` makes much more sense going this route, which I will try to add in a separate PR soon (this weekend).

Reply:
Fixed in 0776441f5060f10767cdd773fc5961c252a024c0 .Marking this resolved

Reply:
I don't think I understand your answer but I'm fine with the rework.

Regarding your next PR, I don't see the link with `FlaskPlugin`. Don't bother explaining it to me if you find it easier to write the PR showing what you meant.

Reply:
@lafrech Sure, adding PR with tests will make much more sense.